src/main/java/org/terasology/launcher/gui/javafx/LogView.java

```diff
@@ -0,0 +1,58 @@
+package org.terasology.launcher.gui.javafx;
+
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import javafx.beans.property.ReadOnlyObjectWrapper;
+import javafx.collections.ObservableList;
+import javafx.scene.control.TableCell;
+import javafx.scene.control.TableColumn;
+import javafx.scene.control.TableView;
+
+import java.text.DateFormat;
+import java.util.Date;
+
+public class LogView {
+
+    private LogView() {}
+
+    public static void updateLogView(TableView<ILoggingEvent> table, ObservableList<ILoggingEvent> log) {
+        table.setItems(log);
+
+        table.setEditable(false);
+        table.setColumnResizePolicy(TableView.CONSTRAINED_RESIZE_POLICY);
+
+        TableColumn<ILoggingEvent, Date> timestampColumn = new TableColumn<>("Timestamp");
+        timestampColumn.setCellValueFactory(item -> new ReadOnlyObjectWrapper<>(new Date(item.getValue().getTimeStamp())));
+        // Fix the width such that `CONSTRAINED_RESIZE_POLICY` only applies to the last column
+        // TODO: setting fixed width is bad and might conflict with system settings such as font size
+        setFixedWidth(timestampColumn, 120);
```

Comment:
Actually, even 120 doesn't work for me. Although this fitting size is dependent on the system font, I find **175** to work fine on my setup. :+1: